src/game-mode.js

```diff
@@ -0,0 +1,207 @@
+          clip();
+          field.append(correctImg);
+          right.play();
+          resolve();
+        } else {
+          setNumberOfMistakes(name);
+          countWrong++;          
+          let field = document.querySelector("#answers");
+          field.classList.add("show");          
+          let wrongImg = createImg("img", "./assets/images/wrong.png", "wrong");
+          clip();
+          FRAGMENT.append(wrongImg);
+          field.append(FRAGMENT);
+          wrong.play();
+        }
+      });
+      result.then(() => {
+        name = playMode(index, arr, countCorrect, countWrong);
+      });
+    });
+  });
+}
+
+function clip() {
+  let answContainer = document.querySelector("#answers");
+  let elements = answContainer.childNodes;
+  let value = elements.length;
+  let containerVolume = 5;  
+  if (value > containerVolume) {
+    elements[0].remove();
+  }
+}
+
+export function playMode(index, arr, countCorrect, countWrong) {
+  if (index === arr.length) {
+    stateOfGame = "end";
+    if (countWrong === 0 && stateOfGame === "end") {
+      let value = "success";
+      showResult(value);
+    } else {
+      let value = "failure";
+      showResult(value);
+    }
+  }
+  if (arr[index] != undefined) {
+    let name = repeatTrack(index, arr);
+    return name;
+  } else {
+    return;
+  }
+}
+export function repeatTrack(index, arr) {
+  let btnRepeat = document.querySelector(".repeat");
+  let element = arr[index];  
+  let start = Object.entries(element);
+  let name = start[0][0];  
+  let sound = Object.values(element)[0];
+  let track = new Audio(sound);  
+  track.play();
+  repeatTrackValue = track;  
+  btnRepeat.addEventListener("click", (event) => {
+    repeatTrackValue.play();
+  });
+  return name;
+}
+
+export function showResult(value) {
+  let content = document.querySelector(".content");
+  let answers = document.querySelector("#answers");
+  answers.classList.remove("show");
+  answers.innerHTML = "";
+  content.innerHTML = "";
+  let resultImg;
+  let resultAudio;
+  if (value === "success") {
+    resultImg = createImg("img", "./assets/images/success.png", "success");
+    resultImg.classList.add("game_end");
+    resultAudio = new Audio("./assets/audio/success.mp3");
+    resultAudio.play();
+    content.append(resultImg);
+    let btnRepeat = document.querySelector(".repeat");
+    if (btnRepeat != null) {
+      btnRepeat.removeEventListener("click", (event) => {
+        repeatTrackValue.play();
+      });
+      btnRepeat.remove();
+    }
+    state.setAttribute("disabled", "disabled");
+    setTimeout(function () {
+      document.location.href = "";
+    }, 3000);
+  } else if (value === "failure") {
+    resultImg = createImg("img", "./assets/images/failure.png", "failure");
+    resultImg.classList.add("game_end");
+    resultAudio = new Audio("./assets/audio/failure.mp3");
+    resultAudio.play();
+    content.append(resultImg);
+    let message = "wrong" + " " + `${countWrong}`;
```

Comment:
A bit messy line of code, should it be like: 
```suggestion
    let message = `wrong ${countWrong}`;
```